Directory.Build.props

```diff
@@ -42,6 +44,11 @@
     <MvcTestingTargets>$(MSBuildThisFileDirectory)src\Mvc\Mvc.Testing\src\build\netstandard2.0\Microsoft.AspNetCore.Mvc.Testing.targets</MvcTestingTargets>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <!-- Publish.targets & SharedFramework.wixproj both use this property, so putting it here in a common location -->
```

Comment:
- Please correct this comment. Publish.targets doesn't use this property.
- nit: Probably better to set the property in SharedFramework.wixproj near where `$(InstallersOutputPath)` is set